MAUI/TreeMap/leaf-item-settings.md

```diff
@@ -0,0 +1,551 @@
+---
+layout: post
+title: Leaf item settings in .NET MAUI TreeMap | Syncfusion
+description: Learn how to harness Syncfusion .NET MAUI TreeMap(STreeMap) control leaf item settings for precise fill color customization, enhancing visual appeal.
+platform: maui
+control: TreeMap (SfTreeMap)
+documentation: ug
+---
+ 
+# TreeMap Leaf item settings
+The `LabelPath` property within the `TreeMapLeafItemSettings` is used to display text for the leaf items. This property allows you to specify the path to the data property containing the text you want to display. The `TreeMapLeafItemSettings` also allows you to customize the spacing between items, stroke color, stroke width, text style, and text format option in the TreeMap control.
```

Comment:
A leaf item represents a visualized data element in the TreeMap. It does not contain child items but may have a parent item if levels are specified in the TreeMap control.